src/quo2/components/selectors/styles.cljs

```diff
@@ -1,130 +1,175 @@
 (ns quo2.components.selectors.styles
```

Comment:
This namespace has changed a lot, since the original `get-color` function was too general and some colors weren't applied according to the designs (probably the designs have changed(?)).

I did my best to abstract and refactor common colors (they are a lot), I also split the styles into multiple functions, the code is bigger now, but I hope it's still clear in case we need to revisit it in the future.